rsyslog/init.sls

```diff
@@ -11,21 +11,42 @@ stoplogger_{{logger}}:
 {% endfor %}
 {% endif %}
 
+remove_old_rsyslog:
+  pkg.purged:
+    - version: 8.16.0-1ubuntu3.1
+
 rsyslog:
+  pkgrepo.managed:
+    - name: deb http://download.opensuse.org/repositories/home:/rgerhards/xUbuntu_16.04/ /
```

Comment:
Just curious why we're using downloading a ubuntu package from a opensuse repository? It seems like the Rsyslog official page links to some: https://www.rsyslog.com/ubuntu-repository/

Reply:
the official repo doesn't include the `omhttp` package unfortunately. Though this opensuse repo is made by the head maintainer of rsyslog

Reply:
hah got it - that's annoying.